client/src/pages/private/SiteParticipantsTable.js

```diff
@@ -0,0 +1,105 @@
+import React, { useEffect, useState } from 'react';
+import _orderBy from 'lodash/orderBy';
+import Grid from '@material-ui/core/Grid';
+import { Box } from '@material-ui/core';
+import store from 'store';
+import { Table } from '../../components/generic';
+
+const columns = [
+  { id: 'participantId', name: 'ID' },
+  { id: 'pptName', name: 'Name' },
+  { id: 'status', name: 'Status' },
+  { id: 'emailAddress', name: 'Email Address' },
+  { id: 'phoneNumber', name: 'Phone Number' },
```

Comment:
We don't need these three fields - might want to check the ticket's table column list again :)

Reply:
Oops! I was looking at the wireframe instead of the ticket :$